content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Accessing HPE Ezmeral Data Fabric Object Storage from Spring Boot S3
+  Micro Service deployed in K3S cluster
```

Comment:
K3s Cluster (lower case "s") [I have seen numerous spellings on the internet, but this appears to be the most official]

Reply:
changed the occurrences to lower case "s"